AppCenter/AppCenter/Internals/Storage/MSLogDBStorage.m

```diff
@@ -67,6 +67,17 @@ - (BOOL)saveLog:(id<MSLog>)log withGroupId:(NSString *)groupId flags:(MSFlags)fl
                                    targetKey ? [NSString stringWithFormat:@"'%@'", targetKey] : @"NULL", (unsigned int)persistenceFlags];
   }
   return [self executeQueryUsingBlock:^int(void *db) {
+           // Check maximum size.
+           NSArray<NSArray *> *rows = [MSDBStorage executeSelectionQuery:@"PRAGMA max_page_count;" inOpenedDatabase:db];
+           NSUInteger maxSize = [(NSNumber *)rows[0][0] intValue] * kMSDefaultDatabaseSizeInBytes;
```

Comment:
`kMSDefaultDatabaseSizeInBytes` is default database size, not page size. We need to find a way to use the `kMSDefaultPageSizeInBytes` which was removed by https://github.com/Microsoft/AppCenter-SDK-Apple/pull/1209.

Reply:
we can use `self.page` here

Reply:
addressed my own feedback by https://github.com/Microsoft/AppCenter-SDK-Apple/pull/1204/commits/9f00eaecc3b9dceaf12a596d25e5e2207158e844